packages/polar/src/lib/deploy/contract.ts

```diff
@@ -259,16 +261,25 @@ export class Contract {
     const wasmFileContent: Buffer = fs.readFileSync(this.contractPath);
 
     const signingClient = await getSigningClient(this.env.network, accountVal);
-    const uploadReceipt = await signingClient.upload(
-      wasmFileContent,
-      {},
-      `upload ${this.contractName}`,
-      customFees
+    const uploadReceipt = await signingClient.tx.compute.storeCode(
+      {
+        sender: accountVal.address,
+        wasmByteCode: wasmFileContent,
+        source: "",
+        builder: ""
```

Comment:
Can we fill some value here? Will make the upload verbose.

Reply:
can you suggest something ?

Reply:
We can pass `builder` and `source` as optional args to `deploy()` and pass them further here. Ref: https://secretjs.scrt.network/interfaces/MsgStoreCodeParams#source